src/client/wire_protocol/header.rs

```diff
@@ -102,20 +107,54 @@ impl Header {
         Header::new(message_length, request_id, 0, op_code)
     }
 
+    /// Construcs a new Header for a reply.
+    ///
+    /// # Arguments
+    ///
+    /// `message_length` - The length of the message in bytes.
+    /// `request_id` - Identifier for the request, or `0` if the the message
```

Comment:
Looks like `request_id` is lined up with `op_code` below though, let's tweak that for consistency.